leases/models.py

```diff
@@ -169,6 +169,29 @@ def __str__(self):
         )
 
 
+class WinterStorageLeaseManager(models.Manager):
+    def get_queryset(self):
+        current_season_start = calculate_winter_storage_lease_start_date()
```

Comment:
should this use `calculate_winter_season_start_date`?

Reply:
hm, seems like it does not matter for the logic we have 🤔

Reply:
Yeah, I think for this case it would be the same.